### DISCUSSION TITLE
time series cross validation and forecast origins

### DISCUSSION THREAD

**Discussion by nickcox896** [OTexts/Discussion#156] [General]
I have a monthly series from 2001m1-2026m7 and  I want to make annual forecasts for the next three years (h=36). I use the time series CV as given here [link](https://otexts.com/fpp2/accuracy.html)  to compute the RMSEs.

In this context, I am not sure where my first forecast origin should begin. 

1) should I use, say, the first 70-80% of initial observations( that is approx 2001m1-2021m5) for model estimation and then start the CV for the next 36 months?
2) should I go back say 4  years (that is approx 2022m7) and then start the CV for the next 36 months?

Is there any suggestion  that I could follow?

**Comment by robjhyndman** [comment]
The initial training set needs to be as small as possible while still being able to reliably estimate the model. That way, you get the most test sets to average over, and therefore get a better measure of accuracy. With monthly data, you can normally fit a model relatively easily with 3 years of data. But if you need more for a sensible fit, adjust this as appropriate.

**Comment by nickcox896** [comment]
Thank you very much. Assuming that I use the first 3 years for the estimation of the model, then what h should I consider for CV RMSEs? Should  obtain these RMSEs for h=12, 24, 36,42,... (that is every 12 months)?

**Comment by robjhyndman** [comment]
Choose whatever horizons for which you want to measure accuracy
